session/bootstrap.go

```diff
@@ -1593,7 +1593,7 @@ func upgradeToVer64(s Session, ver int64) {
 	if ver >= version64 {
 		return
 	}
-	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN `Repl_slave_priv` ENUM('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Execute_priv`", infoschema.ErrColumnExists)
+	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN `Repl_slave_priv` ENUM('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Create_Tablespace_Priv`", infoschema.ErrColumnExists)
```

Comment:
This will only fix installs for those who have not run bootstrap version 64 yet. Can you introduce a new bootstrap version to detect if this historical corruption exists and fix it?